### DISCUSSION TITLE
Washer/Dryer Controls

### DISCUSSION THREAD

**Discussion by plongstock** [nVuln/homebridge-lg-thinq#216] [Q&A]
How do you turn your washer and dryer on/off with this plugin?

**Comment by town3r** [comment]
~~You can’t, only see time remaining.~~

**Comment by nVuln** [comment]
turn on/off feature only worked if your washer running on thinq 1
however, if your washer using thinq 2 and you can turn it on via LG ThinQ app, please take some screenshot and homebridge debug logging, let me see if I can do anything

**Comment by plongstock** [comment]
I am using ThinQ 1.
Thank you!

On Mon, Feb 13, 2023 at 3:01 AM nVuln ***@***.***> wrote:

> turn on/off feature on worked if your washer running on thinq 1
> however, if your washer using thinq 2 and you can turn it on via LG ThinQ
> app, please take some screenshot and homebridge debug logging, let me see
> if I can do anything
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nVuln/homebridge-lg-thinq/discussions/216#discussioncomment-4955595>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A53KHLVWR3PF35I7MUD3WOTWXHZ5ZANCNFSM6AAAAAAUZTJRH4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 
Lisa Rhoades
